Magic-xpa.gitignore

```diff
@@ -0,0 +1,15 @@
+# .gitignore for Magic UniPaaS/xpa (renamed between version 2.0 and 2.1)
```

Comment:
Remove the header, the file name should be clear enough to indicate it's purpose.